layouts/default.vue

```diff
@@ -28,25 +30,41 @@
         id="main"
       />
     </main>
-    <PageFooter />
+    <client-only>
```

Comment:
Does the header need to be rendered server side or could we also only render that on the client to improve performance further?